app/views/overrides/taxonomies/_action_buttons.html.erb

```diff
@@ -12,7 +12,7 @@
     display_link_if_authorized(_("Edit"), hash_for_edit_taxonomy_path(taxonomy) ),
     display_link_if_authorized(_("Nest"), hash_for_nest_taxonomy_path(taxonomy) ),
     display_link_if_authorized(_("Clone"), hash_for_clone_taxonomy_path(taxonomy) ),
-    display_delete_if_authorized(hash_for_taxonomy_path(taxonomy), :confirm => _("Delete %s?") % taxonomy.name, :action => :destroy),
+    display_delete_if_authorized(hash_for_taxonomy_path(taxonomy), :data => { :confirm => _("%s %s has %s Hosts and %s Hostgroups that will need to be reassociated post deletion. Delete %s?") % [taxonomy_title, taxonomy.name, taxonomy.hosts.count, taxonomy.hostgroups.count, taxonomy.name]}, :action => :destroy),
```

Comment:
should these be taxonomy.unscoped.hosts (same for hostgroups).  I'm seeing 0 unless i switch to any context